rgo/schema.py

```diff
@@ -81,6 +81,7 @@ def make_object(self, data, **kwargs):
 
 class ComponentSchema(Schema):
     name = String(required=True)
+    version_from = String()
```

Comment:
The convention seems to be to use `-` instead of `_` as word separator in identifiers, e.g. `spec-path` and `latest-tag` in `GitSchema`, could use use the same for consistency?

Reply:
Changed.